src/components/navbar/navbar.story.mdx

```diff
@@ -7,11 +7,17 @@ import CommonProps from '../../../.storybook/common-props';
 
 # Navbar
 
-A colored message block that can be used to emphasize content or used as a notification banner. Use `Message.Header`
-to create a header of the block, and `Message.Body` for the body of the block.
-
-The style of the message change depending if the Header is included or not, Play with the Controls in the canvas to
- see how this component behave with different props
+The navbar component is a responsive and versatile horizontal navigation bar with the following structure:
+
+- `<Navbar/>`: The main container
+  - `<Navbar.Brand/>` the **left side**, always visible, which usually contains the logo and optionally some links or icons
+    - `<Navbar.Burger/>` the **hamburger** icon, which toggles the navbar menu on touch devices
```

Comment:
Line twelve is formatted differently than every other line. Might be nice to just keep it consistent with the rest ;)